src/test/java/seedu/address/testutil/TypicalGuests.java

```diff
@@ -24,10 +24,10 @@
     public static final Guest ALICE = new GuestBuilder().withName("Alice Pauline")
             .withEmail("alice@example.com")
             .withPhone("94351253")
-            .withTags("vip").build();
+            .withTags(TypicalTags.GUEST_TAG_VIP.tagName).build();
```

Comment:
This currying is quite long. A suggestion is to edit the `withTags` constructor in future, to accept `Tag`s or a `String`.

Reply:
What currying?? There's no currying here.